src/components/Table/GridTable.test.tsx

```diff
@@ -559,6 +572,18 @@ describe("GridTable", () => {
       expect(row(r, 4).getAttribute("data-render")).toEqual("1");
       expect(row(r, 5).getAttribute("data-render")).toEqual("1");
       expect(row(r, 6).getAttribute("data-render")).toEqual("1");
+
+      // And the table re-renders for some other reason
+      click(r.rerenderParent);
+
+      // Then memoization did not break
+      expect(row(r, 0).getAttribute("data-render")).toEqual("2");
```

Comment:
question: Why did the header rerender?

Reply:
`sortState` changed, and right now we don't have cell-level re-rendering of the sort headers. Just the whole row re-renders.